bsip-0043.md

```diff
@@ -0,0 +1,96 @@
+    BSIP: 0043
+    Title: Market fee sharing
+    Authors: OpenLedgerApp <https://github.com/OpenLedgerApp>
+    Status: Draft
+    Type: Protocol
+    Created: 2018-08-23
+    Discussion: https://github.com/bitshares/bsips/issues/102
+    Worker:
+
+
+# Abstract
+
+When creating a new asset, the asset issuer is the only beneficiary of the market fees in the current implementation. And the only way to affect the asset community growth is the market fee percentage. For example, one can decrease the market fee and it will result in somewhat larger number of trades with this asset. In this way the asset issuer might get a bigger profit during to increasing the trade volume with this asset.
```

Comment:
Certainly not "the only way". I doubt that global BTC trade volume depends on market fees alone.

Reply:
You're right, it's not the only way, but one of the ways